lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -60,6 +62,8 @@ function CFCRandomSpawn.refreshMapInfo()
     CENTER_UPDATE_INTERVAL = customSpawnConfigForMap.centerUpdateInterval or CFCRandomSpawn.Config.CENTER_UPDATE_INTERVAL
     IGNORE_BUILDERS = CFCRandomSpawn.Config.IGNORE_BUILDERS
 
+    CFCRandomSpawn.spawnsUnderARoof = nil
```

Comment:
what's the point of this?

Reply:
Purges it since it might be now be outdated